README.md

```diff
@@ -145,6 +145,7 @@ ios:
   key_type: "pem" # could be pem, p12 or p8 type
   password: "" # certificate password, default as empty string.
   production: false
+  max_concurrent_pushes: 100
```

Comment:
Can you add comment `just for push ios notification`?

Reply:
We can help @McPo with suggestions like this:
```suggestion
  max_concurrent_pushes: 100 # just for push ios notification
```